src/utils/convert-filter.js

```diff
@@ -9,10 +9,16 @@ const convertFilter = (filter) => {
     const { property, value } = filterProperty
     switch (property.type()) {
     case 'string':
-      return {
-        [property.name()]: { [Op.like]: `%${escape(value)}%` },
-        ...memo,
-      }
+        if (property.sequelizePath.values) {
```

Comment:
I faced the same issue and I was working on a similar solution as well.
My condition here was:
```
property.sequelizePath.type instanceof DataTypes.ENUM
```
But I guess your condition is even better!